frameworks/hdfs/tests/test_shakedown.py

```diff
@@ -68,7 +58,7 @@ def test_integrity_on_name_node_failure():
     This test checks that it is possible to write and read data after the first name node fails.
     """
     tasks.kill_task_with_pattern("NameNode", 'name-0-node.hdfs.autoip.dcos.thisdcos.directory')
-    time.sleep(1)  # give NameNode a chance to die
+    time.sleep(9)  # give NameNode a chance to die
```

Comment:
I can change it back, 1 second seemed pretty aggressive though given all the other times in the tests.